### PR TITLE
Add debug option to rkt task config

### PR DESCRIPTION
**Pull request by nak3**

### CODE REVIEW

File: client/driver/rkt.go [hunk 1]
```diff
@@ -62,6 +62,7 @@ type RktDriverConfig struct {
 	TrustPrefix      string   `mapstructure:"trust_prefix"`
 	DNSServers       []string `mapstructure:"dns_servers"`        // DNS Server for containers
 	DNSSearchDomains []string `mapstructure:"dns_search_domains"` // DNS Search domains for containers
+	debug            string   `mapstructure:"debug"`
```
Comment (diptanu) [inline client/driver/rkt.go]: Comment and make it `Debug`

File: client/driver/rkt.go [hunk 2]
```diff
@@ -190,12 +194,16 @@ func (d *RktDriver) Start(ctx *ExecContext, task *structs.Task) (DriverHandle, e
 	// Build the command.
 	var cmdArgs []string
 
+	// Add debug option to rkt command.
+	debug := "false"
+	debug, _ = task.Config["debug"].(string)
```
Comment (diptanu) [inline client/driver/rkt.go]: Use `strings.FormatBool`

**Comment by nak3** [comment]
Thank you! I updated them.

File: client/driver/rkt.go [hunk 3]
```diff
@@ -62,6 +63,7 @@ type RktDriverConfig struct {
 	TrustPrefix      string   `mapstructure:"trust_prefix"`
 	DNSServers       []string `mapstructure:"dns_servers"`        // DNS Server for containers
 	DNSSearchDomains []string `mapstructure:"dns_search_domains"` // DNS Search domains for containers
+	debug            bool     `mapstructure:"debug"`              // Enable debug option for rkt command
```
Comment (diptanu) [inline client/driver/rkt.go]: Make it `Debug`

File: client/driver/rkt.go [hunk 4]
```diff
@@ -190,12 +195,17 @@ func (d *RktDriver) Start(ctx *ExecContext, task *structs.Task) (DriverHandle, e
 	// Build the command.
 	var cmdArgs []string
 
+	// Add debug option to rkt command.
```
Comment (diptanu) [inline client/driver/rkt.go]: Since the type of `Debug` is bool, we don't need any of this now.

File: client/driver/rkt.go [hunk 5]
```diff
@@ -190,12 +195,17 @@ func (d *RktDriver) Start(ctx *ExecContext, task *structs.Task) (DriverHandle, e
...
+	debug := false
+	debug, _ = task.Config["debug"].(bool)
+	sdebug := strconv.FormatBool(debug)
```
Comment (diptanu) [inline client/driver/rkt.go]: Since the type of `Debug` is bool, we don't need any of this now.

**Pull request merged**